deno.jsonc

```diff
@@ -7,7 +7,6 @@
         "dev": "deno fmt && deno lint && deno task test && deno task check",
         "coverage": "rm -rf ./test/cov_profile && deno task test --coverage=./test/cov_profile && deno coverage --lcov --output=./coverage.lcov ./test/cov_profile",
         "report": "genhtml ./coverage.lcov --output-directory ./test/coverage/ && echo 'Point your browser to test/coverage/index.html to see the test coverage report.'",
-        "bundle": "cd bundling && ./bundle-all.sh",
```

Comment:
Why did you remove the bundling?

Reply:
You're right, I misread, I was thinking about `bundle-web`. This can indeed be removed, thanks!